service/src/test/java/uk/gov/hmcts/reform/fpl/service/ccd/CoreCaseDataServiceTest.java

```diff
@@ -0,0 +1,73 @@
+package uk.gov.hmcts.reform.fpl.service.ccd;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+import uk.gov.hmcts.reform.authorisation.generators.AuthTokenGenerator;
+import uk.gov.hmcts.reform.ccd.client.CoreCaseDataApi;
+import uk.gov.hmcts.reform.ccd.client.model.CaseDataContent;
+import uk.gov.hmcts.reform.ccd.client.model.Event;
+import uk.gov.hmcts.reform.ccd.client.model.StartEventResponse;
+import uk.gov.hmcts.reform.fpl.config.SystemUpdateUserConfiguration;
+import uk.gov.hmcts.reform.idam.client.IdamClient;
+import uk.gov.hmcts.reform.idam.client.models.UserDetails;
+
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+import static uk.gov.hmcts.reform.fpl.CaseDefinitionConstants.CASE_TYPE;
+import static uk.gov.hmcts.reform.fpl.CaseDefinitionConstants.JURISDICTION;
+
+@ExtendWith(SpringExtension.class)
+@ContextConfiguration(classes = {CoreCaseDataService.class})
+class CoreCaseDataServiceTest {
+    @MockBean
+    private SystemUpdateUserConfiguration userConfig;
+    @MockBean
+    private AuthTokenGenerator authTokenGenerator;
+    @MockBean
+    private IdamClient idamClient;
+    @MockBean
+    private CoreCaseDataApi coreCaseDataApi;
+
+    @Autowired
+    private CoreCaseDataService service;
```

Comment:
This is not consistent with other tests where we use @Mock an @InjectMocks, but no biggie for me.

Reply:
Alright, changed it to use Mockito only in unit tests.